### PR TITLE
Add Nextiva App Stable

### PR DESCRIPTION
**Pull request by scottb**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/nextiva.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'nextiva' do
+  version '21.3.0.1311'
+  sha256 '1e5de7747db38e99cdc76e6b6194d24f44078d89eacc88843250a8fa13ce1466'
+
+  # dm.nextiva.com was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/nextiva.rb]: Irrelevant and needs to be removed. Comment is only necessary when download host differs from homepage.

File: Casks/nextiva.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  url "https://dm.nextiva.com/dms/bc/pc/Nextiva_App.bc-uc.osx-#{version}.dmg"
+  name 'Nextiva'
+  name 'Nextiva App'
```
Comment (vitorgalvao) [inline Casks/nextiva.rb]: Remove this one.

File: Casks/nextiva.rb (hunk 1 above)
Comment (scottb) [inline Casks/nextiva.rb]: The download host is "dm.nextiva.com", the homepage is "www.nextiva.com" -- those differ. But, ok, I'll remove it. Perhaps the "add a cask" documentation should be clarified?

Comment (vitorgalvao) [inline Casks/nextiva.rb]: `nextiva.com` is pretty clear as a host. This is not a case of a suspicious subdomain. The documentation does not need clarification on this point.

File: Casks/nextiva.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cask 'nextiva' do
+  version '21.3.0.1311'
+  sha256 '1e5de7747db38e99cdc76e6b6194d24f44078d89eacc88843250a8fa13ce1466'
+
+  url "https://dm.nextiva.com/dms/bc/pc/Nextiva_App.bc-uc.osx-#{version}.dmg"
+  name 'Nextiva'
+  name 'Nextiva App'
```
Comment (vitorgalvao) [inline Casks/nextiva.rb]: Still need to remove this second one.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in b3fb5e0c7cbf83d1d8934a96531e77cf9d2669d4. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

File: Casks/nextiva.rb (hunk 1 above)
Comment (scottb) [inline Casks/nextiva.rb]: You've sort of hit on exactly the problem with the documentation. The _host_ is `dm.nextiva.com`, while `nextiva.com` is simply a part of its domain name. While the second-level domain name typically corresponds to the administrative authority, they're not necessarily the same, and the documentation actually says the comment must be added "when URL and Homepage hostnames differ".

Comment (vitorgalvao) [inline Casks/nextiva.rb]: You are correct. The documentation does need clarification, then.

Care to submit a PR and we take it from there?

**Pull request closed without merging**